src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -262,11 +262,12 @@ public enum LegendMaker implements LogicCardInfo {
           text "Count the number of React Energy cards attached to Cradily and choose up to that number of your opponent's Evolved Pokémon. Remove the highest Stage Evolution card from each of those Pokémon, then have your opponent shuffle those cards into his or her deck."
           energyCost C, C
           attackRequirement {
-            assert self.cards.findAll {it.name.contains("React Energy")} : "No React Energies attached to this Pokemon"
-            assert opp.all.findAll{ it.evolution } : "Your opponent has no evolved pokemon"
+            assert self.cards.any{it.name.contains("React Energy")} : "No React Energies attached to this Pokemon"
+            assert opp.all.any{ it.evolution } : "Your opponent has no evolved pokemon"
           }
           onAttack {
```

Comment:
Looks like this is moving the cards to the hand? Should be shuffled into deck according to attack text.

Reply:
Yup, left this one as a TODO before leaving for bed a couple days ago. Will look at it in the morning, should do a full rewrite of it.

Reply:
This one I wanted to be able to test it, so is already commited in master: https://github.com/axpendix/tcgone-engine-contrib/compare/d087369a3bddc17ec62b2189feac1eff4339f2f7...2177bdbc0453d9cf1c972ba125fd4816738e49bf

Now shuffles to deck and is more user-friendly regarding selection.

Reply:
You forgot to shuffle the deck in that commit it seems, but aside from that, all looks good to me now.